modular_skyrat/modules/goofsec/jobs/sol_fed.dm

```diff
@@ -1,7 +1,11 @@
 GLOBAL_VAR_INIT(amt_911_responders, 0)
 GLOBAL_VAR_INIT(votes_for_bad_911_call, 0)
+GLOBAL_VAR_INIT(amt_SWAT_responders, 0)
+GLOBAL_VAR_INIT(votes_for_treason, 0)
 GLOBAL_VAR_INIT(fradulent_911_declared, FALSE)
+GLOBAL_VAR_INIT(treason_declared, FALSE)
 GLOBAL_VAR(caller_of_911)
+GLOBAL_VAR(call_911_msg)
```

Comment:
Can we really not do without **9** global variables? I'm sure there's a better way to do this.

Reply:
These variables gotta go somewhere and they're in a scope that requires them to be global. It's that or stick them on a subsystem.

Reply:
Surely there's another way around it.

Reply:
It's either storing these all in one mega glob, storing them on a subsystem, or list looping shenanigans.